### ISSUE TITLE
Using react-color with reactCSS 0.4 significantly increases bundle size

### ISSUE THREAD

**Issue by dht** [casesandberg/react-color#199]
This is because React and Lodash are added twice, right?
![screen shot 2016-08-16 at 1 00 49 pm](https://cloud.githubusercontent.com/assets/137832/17696624/4912debc-63b8-11e6-9cf5-b0825e763f40.png)

**Comment by transcranial** [comment]
Definitely. It's pretty hefty. All of lodash 3 and react 0.14 gets thrown in there as most are probably using modularized lodash 4 and react 15.

```
├─┬ react-color@2.2.2
│ ├── lodash.isplainobject@4.0.6
│ ├── lodash.throttle@4.1.1
│ ├── material-colors@1.0.0
│ ├── merge@1.2.0
│ ├── react-addons-shallow-compare@15.3.0
│ ├─┬ reactcss@0.4.6
│ │ ├── lodash@3.10.1
│ │ └─┬ react@0.14.8
```

Would be great if reactcss can be upgraded!

**Comment by casesandberg** [comment]
Fixed! `react-color@2.2.3`
